File: hw/collahuazo/u3/hw24CompositePaterns/CompositePatternJS/composite/model/Clerk.js
const { Employee } = require("./Employee.js");
class Clerk extends Employee {
  constructor(aName) {
    super();
    this.title = "Clerk";
    this.name = aName;
  }

  stateName() {
    super.stateName();
  }
}

module.exports = { Clerk };

File: hw/basurto/u3/hw24CompositePattern/hw24CompossitePattern.java/CompossitePattern/src/main/java/ec/edu/espe/CompositePattern/model/Clerk.java
package ec.edu.espe.CompositePattern.model;

 public class Clerk extends Employee {
    public Clerk(String aName) {
        this();
        name = aName;
    }

    public Clerk() {
        title = "Clerk";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}